blockbase/block-templates/index.html

```diff
@@ -6,6 +6,7 @@
 	<!-- wp:post-title {"isLink":true} /-->
 	<!-- wp:post-featured-image {"isLink":true} /-->
 	<!-- wp:post-excerpt /-->
+	<!-- wp:spacer {"height":40} -->
```

Comment:
The spacer block needs to have the HTML stuff inside of it and get closed off to work properly.  I usually just instantiate the block in the block editor and copy/paste 'cause I usually forget how it goes. :P

```
	<!-- wp:spacer {"height":40} -->
	<div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
	<!-- /wp:spacer -->